mtgjson5/__main__.py

```diff
@@ -36,9 +22,17 @@ def build_mtgjson_sets(
     """
     Build each set one-by-one and output them to a file
     :param sets_to_build: Sets to construct
-    :param output_pretty: Should we dump minified?
-    :param include_referrals: Should we include referrals?
+    :param output_pretty: Should we dump minified
+    :param include_referrals: Should we include referrals
```

Comment:
id leave the ? at the end (or at least drop them in a separate commit)

Reply:
Moved to a commit